logstash-output-elasticsearch.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'stud', ['>= 0.0.17', '~> 0.0']
   s.add_runtime_dependency 'cabin', ['~> 0.6']
-  s.add_runtime_dependency "logstash-core-plugin-api", "~> 2.0"
+  s.add_runtime_dependency "logstash-core-plugin-api", [">= 2.0", "< 3.0"]
```

Comment:
Why this change here? IMO, we should do this as a mass update for all plugins. Otherwise, PH's script will be confused.